.gitignore

```diff
@@ -19,7 +19,7 @@ jsdoc/
 .cache
 /packages/**/node_modules
 /tools/internal-rules/node_modules
-/.vscode
+# /.vscode
```

Comment:
[Question] Is there any reason to keep this around? Since `.vscode/settings.json` is now intentional:

```suggestion
```